public/docs/ts/latest/guide/displaying-data.jade

```diff
@@ -119,59 +117,54 @@ figure.image-display
   :marked
     ## Constructor or variable initialization?
 
-    We initialized our component properties using variable assignment.
-    This is a wonderfully concise and compact technique.
-
-    Some folks prefer to declare the properties and initialize them within a constructor like this:
+    You initialized your component properties using variable assignment. You could have
```

Comment:
I'm not crazy about "you" here, since they didn't make the decision. Can you think of a better way to phrase this?